src/scripts.js

```diff
@@ -13,29 +14,66 @@ import Hydration from './Hydration';
 import Sleep from './Sleep';
 
 
+let hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData').then(function(response) {
+  return response.json()
+});
+let activityData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData').then(function(response) {
+  return response.json()
+});
+let sleepData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData').then(function(response) {
+  return response.json()
+});
+let userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData').then(function(response) {
+  return response.json()
+});
+let combinedData = {
```

Comment:
I never thought of making this sort of object structure but it looks really clean